test/integration-test.py

```diff
@@ -241,6 +248,45 @@ def run_client_tests():
     cmd = os.path.join(root, 'tests', 'boulder-integration.sh')
     run_custom(cmd, cwd=root)
 
+def run_expired_authz_purger_test(challenge_types):
+    if run_node_test("challenge-not-started.com", challenge_types[0], None, "startChallenge")\
```

Comment:
Calling run_node_test here is not good. run_node_test is a big function that does a lot of stuff, including OCSP checking, and we're just calling it to take advantage of its subprocess.Popen call to test.js, most of which we don't care about. Instead, let's call subprocess.Popen ourselves, with a much-reduced set of options (since we don't care where any of the keys or certs go).